src/components/Personal.tsx

```diff
@@ -0,0 +1,90 @@
+import { Fragment, useState } from "react";
+import style from "./Personal.module.css";
+import { TextField } from "@mui/material";
+
+function Personal(props: any) {
+  const [name, setName] = useState("");
+  const [department, setDepartmant] = useState("");
+  const [passport, setPasport] = useState("");
+  const [id, setId] = useState("");
+
+  const onNameHandler = (event: any) => {
+    setName(event.target.value);
+  };
+  const onDepartmentHandler = (event: any) => {
+    setDepartmant(event.target.value);
+  };
+  const onPassportHandler = (event: any) => {
+    setPasport(event.target.value);
+  };
+  const onIdHandler = (event: any) => {
+    setId(event.target.value);
+  };
+
+  const personalData = {
+    nameSurname: name,
+    department: department,
+    passportNo: passport,
+    idNo: id,
+  };
+  props.onSavePersonalData(personalData);
+
+  // setName("");
+  // setDepartmant("");
+  // setId("");
+  // setPasportInput("");
+
+  return (
+    <Fragment>
+      <div className={style.form}>
+        <div className={style.personalContainer}>
+          <h1 className={style.h1}>Personal Data</h1>
```

Comment:
Maybe instead of h1 you can use MUI Typography components? 
https://mui.com/material-ui/api/typography/#main-content
They have same useful props available